### ISSUE TITLE
Download playlist failed

### ISSUE THREAD

**Issue by MrFox360** [scdl-org/scdl#505]
Hy. Please could somebody help me. I can not download playlists. Single track are not a ploblem.
Thanks in advance
I get this traceback:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Scripts\scdl.exe\__main__.py", line 7, in <module>
    sys.exit(main())
             ^^^^^^
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\scdl.py", line 402, in main
    download_url(client, typing.cast(SCDLArgs, python_args))
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\scdl.py", line 582, in download_url
    download_playlist(client, playlist, kwargs)
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\scdl.py", line 739, in download_playlist
    download_track(
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\scdl.py", line 1056, in download_track
    filename, is_already_downloaded = download_hls(
                                      ^^^^^^^^^^^^^
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\scdl.py", line 984, in download_hls
    re_encode_to_out(
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\scdl.py", line 1365, in re_encode_to_out
    encoded = re_encode_to_buffer(
              ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\scdl.py", line 1563, in re_encode_to_buffer
    _add_metadata_to_stream(track, encoded_data, kwargs, playlist_info)
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\scdl.py", line 1331, in _add_metadata_to_stream
    assemble_metadata(mutagen_file, metadata)
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\functools.py", line 909, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\scdl\metadata_assembler.py", line 164, in _
    file["trkn"] = str(meta.album_track_num)
    ~~~~^^^^^^^^
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\mutagen\_file.py", line 74, in __setitem__
    self.tags[key] = value
    ~~~~~~~~~^^^^^
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\mutagen\mp4\__init__.py", line 372, in __setitem__
    self._render(key, value)
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\mutagen\mp4\__init__.py", line 388, in _render
    return render_func(self, key, value, *render_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\fuxiu\AppData\Local\Programs\Python\Python312\Lib\site-packages\mutagen\mp4\__init__.py", line 651, in __render_pair
    track, total = v
    ^^^^^^^^^^^^
ValueError: not enough values to unpack (expected 2, got 1)

**Comment by 7x11x13** [comment]
Can you post the url of the playlist?

**Comment by MrFox360** [comment]
https://soundcloud.com/mrfox1990/sets/liquid-1

thanks for reply

**Comment by HeartbeatingForCenturies** [comment]
> https://soundcloud.com/mrfox1990/sets/liquid-1
> 
> thanks for reply

this playlist doesn't exist, make sure its public

**Comment by MrFox360** [comment]
https://soundcloud.com/mrfox1990/sets/liquid-1?si=69d4cbd5504a40e5a61b79f82687b45c&utm_source=clipboard&utm_medium=text&utm_campaign=social_sharing

Sorry now its public

**Comment by 7x11x13** [comment]
Fixed in v2.11.2

**Comment by MrFox360** [comment]
Oh boy yeah. Thank you
